Ajoute les discours 2012, 2015, 2016 et 2017 lors de la conférence

MattiSG opened the pull request:
@fredZen @egaillot ces documents étaient dans la dropbox, je trouve utile voire important de les archiver publiquement, est-ce ok pour vous ? 🙂

Aussi, si vous savez qui a prononcés ces discours, ça peut être utile à consolider.

MattiSG committed:
Ajoute les discours lors de la conférence

MattiSG committed:
Ajoute le discours 2017

egaillot commented:
@MattiSG : le texte de 2016, c'est moi qui l'ai prononcé. En ce qui me concerne je ne trouve ni utile ni important d'archiver ces textes publiquement, mais si tu penses le contraire, je m'en remets à ton jugement et te laisse le faire.

MattiSG committed:
Ajoute l'orateur 2016

davidbgk commented on _pages/index.md:
```diff
@@ -41,6 +41,17 @@ Le bureau de l'association est actuellement composé de :
+
+{% assign mots = site.mot_conference | reverse %}
+
+{% for mot in mots %}
```
Missing `-` trick to avoid the extra `<p>`

davidbgk commented on _pages/index.md:
```diff
@@ -41,6 +41,17 @@ Le bureau de l'association est actuellement composé de :
+{% assign mots = site.mot_conference | reverse %}
+
+{% for mot in mots %}
+- [Discours lors de la conférence {{ mot.id | remove: "/mot_conference/" | date: "%Y" }}]({{ mot.url }})
```
\+ `par {{ mot.speaker }}` ?

davidbgk commented on _mot_conference/2015-06-18.md:
```diff
@@ -0,0 +1,67 @@
+---
+speaker: Frédéric Merizen
```
The speaker is not rendered within the page, maybe add him (a 👩 this year? 🤷‍♂️) as a signature?

davidbgk reviewed (approved):
With comments

MattiSG committed:
Corrige le balisage

Supprime le trailing whitespace pour assurer un rendu sous la forme d'une <ul><li> correcte

MattiSG committed:
Upstream merge

MattiSG committed:
Améliore les URL des mots de la conférence

MattiSG committed:
Ajoute un titre et l'orateur aux mots de la conf

MattiSG commented:
La demande de revue était très opportuniste, j'ai fait le choix d'intégrer sur la base de la première validation.

J'ai pris en compte toutes les remarques 🙂
